react/components/EditorContainer/StoreEditor/Styles/StyleEditor/typography/FileFontEditor.tsx

```diff
@@ -70,22 +87,26 @@ const FileFontEditor: React.FunctionComponent<FileFontEditorProps> = ({
 
   return (
     <div className="mv6 w-100">
-      <Input
-        size="small"
-        label={
-          <FormattedMessage
-            id="admin/pages.editor.styles.edit.font-family.title"
-            defaultMessage="Font Family"
+      <FormattedMessage
+        id="admin/pages.editor.styles.edit.font-family.title"
+        defaultMessage="Font Family"
+      >
+        {label => (
+          <Input
+            size="small"
+            label={label}
+            value={family}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
You could declare this callback using [`useCallback`](https://reactjs.org/docs/hooks-reference.html#usecallback) to avoid unnecessary re-renders.